features/step_definitions/provider_steps.rb

```diff
@@ -125,15 +125,15 @@ def import_simple_layout(provider)
 end
 
 Given "{provider} has sso {enabled} for all users" do |provider, enabled|
-  provider.settings.update!(enforce_sso: enabled)
+  provider.settings.update_column(:enforce_sso, enabled)
 end
 
 And /^the sso integration is (published|hidden)$/ do |state|
   @authentication_provider.update!(published: state == 'published')
 end
 
 And /^the sso integration is tested$/ do
-  EnforceSSOValidator.any_instance.stubs(:valid?).returns(true).at_most(2)
+  EnforceSSOValidator.any_instance.stubs(:valid?).returns(true).at_most(3)
```

Comment:
maybe `#at_least_once` makes more sense here?

This would show that the condition was tested at least once. I can't imagine why we have an upper limit but no lower limit :)

```suggestion
  EnforceSSOValidator.any_instance.stubs(:valid?).returns(true).at_least_once
```

Reply:
Yeah, probably we don't need that granularity